### PR TITLE
update citation

### PR DESCRIPTION
**Pull request by mdeceglie**
- ~Code changes are covered by tests~
- ~Code changes have been evaluated for compatibility/integration with TrendAnalysis~
- ~New functions added to `__init__.py`~
- ~API.rst is up to date, along with other sphinx docs pages~
- ~Example notebooks are rerun and differences in results scrutinized~
- ~Updated changelog~

### CODE REVIEW

**Comment by mdeceglie** [comment]
I'm not crazy about using the universal DOI link to get to the Zenodo page. Perhaps we could use this: https://zenodo.org/search?page=1&size=20&q=conceptrecid:%221210316%22&sort=-version&all_versions ? Other ideas?

**Comment by mdeceglie** [comment]
#296 is relevant

File: docs/sphinx/source/index.rst [hunk 1]
```diff
@@ -238,27 +238,35 @@ take one of several types, we document them using the type alises listed below:
 
 Citing RdTools
 --------------
+To cite RdTools, please use the following along with the version number
+and the specific DOI coresponding to that version from `zenodo <https://doi.org/10.5281/zenodo.1210316>`_:
```
Comment (kanderso-nrel) [inline docs/sphinx/source/index.rst:242]: Do you want to capitalize Zenodo here too?

**Review by kanderso-nrel — COMMENTED** [review]
> I'm not crazy about using the universal DOI link to get to the Zenodo page. Perhaps we could use this: https://zenodo.org/search?page=1&size=20&q=conceptrecid:%221210316%22&sort=-version&all_versions ?

Works for me.

Thoughts on adding linked DOIs for each of the "underlying workflow" references?  Would be nice to be able to click a link instead of having to search the title. 

- Robust PV degradation: 10.1109/JPHOTOV.2017.2779779
- Quantifying soiling loss: 10.1109/JPHOTOV.2017.2784682
- Overcoming communication outages: 10.1109/PVSC45281.2020.9300635
- Performance comparison of clipping detection: already listed

**Review by kanderso-nrel — APPROVED** [review]

**Pull request merged**